pyproject.toml

```diff
@@ -37,7 +37,7 @@ pysolr = "^3.9.0"
 eutils = ">=0.6.0"
 requests-cache = "^1.0.1"
 click = "*"
-semsimian = "0.1.14"
+semsimian = ">=0.1.14"
```

Comment:
definitely correct? so nothing in this PR relies on a later version

Reply:
This PR relies on the latest version .. 0.1.15

Reply:
OK I will make a new PR where I fix the pyproject